src/hyp3_akfire_safe/__init__.py

```diff
@@ -1,7 +1,9 @@
-"""Plugin to get landsat scenes from an input location."""
+"""Plugin with workflows for akfire-safe."""
 
 from importlib.metadata import version
 
+import sklearn  # noqa
```

Comment:
Why has this been added? I don't see sklearn actually used, nor do I see it added to the pyproject.toml nor the envitonment.yml.